boards/shields/st7789v_generic/doc/index.rst

```diff
@@ -0,0 +1,58 @@
+.. _st7789v_generic:
+
+Generic ST7789V Display Shield
+##############################
+
+Overview
+********
+
+This is a generic shield for display shields based on ST7789V display
+controller. More information about the controller can be found in
+`ST7789V Datasheet_`.
```

Comment:
Underscore (indicating an outgoing link) is after the back-quoted text:
```
`ST7789V Datasheet`_.
```

Reply:
thx for the input!